change var to const

rkathrn opened the pull request:
Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__

<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->
changed 3 instances of var to const

#### Motivation
<!-- ❓ Why are you making this change? Help us understand how your changes help readers. -->
Writing Day - WTD2022

#### Supporting details
<!-- 🔗 Link to supporting information, such as bug trackers, source control, release notes, and vendor docs. -->

#### Related issues
<!-- 🔨 If applicable, use "Fixes #XYZ" -->

#### Metadata
<!-- ✅ Check a box if applicable, like this: [x]

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

rkathrn committed:
change var to const

Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__

wbamberg commented on files/en-us/web/javascript/reference/global_objects/object/__definesetter__/index.md:
```diff
@@ -70,13 +70,13 @@ console.log(o.anotherValue); // 5
 // Using the set operator
-var o = { set value(val) { this.anotherValue = val; } };
+const o = { set value(val) { this.anotherValue = val; } };
 o.value = 5;
 console.log(o.value); // undefined
 console.log(o.anotherValue); // 5
 
 // Using Object.defineProperty
-var o = {};
+const o = {};
```
This looks like the same pattern as https://github.com/mdn/content/pull/16531#discussion_r878934185 , I think?

rkathrn committed:
revision-val/const

broke standard-compliant ways into two js codeblocks to avoid declaring o twice

queengooborg reviewed (approved):
Thanks for the quick updates!
